.github/workflows/Intel_Parallelworks_CI.yaml

```diff
@@ -94,15 +103,15 @@ jobs:
           - d96_500m.solo.mtn_schar.sh
     steps:
       # This will end the slurm job started in the checkout job
-      - run: scancel -n SP_$GITHUB_REF
+      - run: scancel -n $GITHUB_SHA
```

Comment:
Why is this needed?  Shouldn't the checkout slurm job terminate on it's own correctly?

Reply:
I have this specific slurm job running a sleep command for 20 minutes.  If the checkout and build steps finish before 20 minutes, I would like to cancel that job.  It then frees up the nodes to then be assigned to the tests that run next.